third_party/java/arcs/build_defs/internal/manifest.bzl

```diff
@@ -27,6 +42,27 @@ def arcs_manifest(name, srcs, deps = [], visibility = None):
         visibility = visibility,
     )
 
+    if manifest_proto:
```

Comment:
Why can't we use `arcs_manifest_proto` directly ?

Reply:
What do you mean exactly? We are using `arcs_manifest_proto` in line 49.

Are you asking why don't we get the developer to manually invoke the `arcs_manifest_proto` themselves? We could do that, but then we'd have to get them to pass in the label for that into this rule, so that we can refer to the correct file when we create the policy test. 

Ultimately I thought this:

```
arcs_manifest(
  name = "foo",
  srcs = ["foo.arcs"],
  manifest_proto = True,
  policy_test = True,
  policy_options = "options.textproto",
)
```

was nicer than this:

```
arcs_manifest(
  name = "foo",
  srcs = ["foo.arcs"],
  manifest_proto = ":foo_proto",
  policy_test = True,
  policy_options = "options.textproto",
)

arcs_manifest_proto(
  name = "foo_proto",
  srcs = ["foo.arcs"],
)
```

Reply:
My question was more about why we needed to make it part of `arcs_manifest`? Is it so that we can package the binary proto if needed?

Reply:
It's just to make it simpler for the developer. It's not to package it up, it's just so that they don't have to generate it themselves. If you want the binary proto you need to dep on either `:foo_manifest_proto` or `:foo.bin.pb`, you can't just dep on `:foo`, that will only give you the .arcs file.

We could add the .bin.pb file to the `:foo` filegroup if we wanted (that would let you package it up automatically), but I'm not sure that we do.